packages/pharos/src/components/icon/storyArgs.ts

```diff
@@ -3,6 +3,8 @@ import { iconNames } from '../../utils/iconNames';
 export const defaultArgs = {
   name: 'checkmark',
   description: '',
+  a11yTitle: 'This is the icon title',
```

Comment:
This does give the example a way of indicating how the a11yTitle is used, but should we include a more relevant title for the purpose of the icon, eg. "checkmark"?